src/Commands/QuantDrushCommands.php

```diff
@@ -70,14 +70,30 @@ private function getDrushPath() {
    * @option threads
    *   Number of threads to use (default 5)
    * @usage quant:run-queue --threads=5
+   * @option unlock
+   *   Allow a queue run even if another run is in progress (default is false)
+   * @usage quant:run-queue --unlock
```

Comment:
See Stu's comments about adding another drush command to be able to remove the lock file.